index.html

```diff
@@ -13,7 +13,8 @@ <h1>Digital Humanities Lab</h1>
 </div>
 
 <!-- temporary warning -->
-{% include warning_banner.html text="All Yale Library websites and resources will be offline from approx. 9 pm EDT July 17 to 4 am EDT July 18." %}
+{% include warning_banner.html text="All Yale Library websites and resources will be offline from 9 pm EDT July 17 to approx. 10:30 am EDT on July 18. Updates 7/18 on <a href="https://twitter.com/YaleLibrary">@YaleLibrary</a> Twitter.
```

Comment:
The outer quotes should be single to avoid the conflicting nested double quotes.

Otherwise, looks good. But we will need to style the link differently (underline?).